ci-operator/step-registry/openshift-qe/udn-bgp/openshift-qe-udn-bgp-commands.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+set -x
+
+SSH_ARGS="-i /bm/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+bastion="$(cat /bm/address)"
+LAB_CLOUD=$(cat ${CLUSTER_PROFILE_DIR}/lab_cloud || cat ${SHARED_DIR}/lab_cloud)
+export LAB_CLOUD
+
+echo "LAB is $LAB_CLOUD"
+LOGIN=password
+
+SCRIPT_FILE="remote_script.sh"
+
+# This script is run on the bastion host. It sshs into the hypervisor creates a VM
+# and then spawns external BGP server on the VM.
+cat > $SCRIPT_FILE << 'EOF'
+#!/bin/bash
+set -x
+LAB_CLOUD=$1
+export KUBECONFIG=/root/vmno/kubeconfig
+LOGIN=password
+INVENTORY_FILE="/root/jetlag/ansible/inventory/$LAB_CLOUD.local"
```

Comment:
I think the inventory file will be on `/tmp`, not on `/root`

Reply:
I have tested all this in alias lab with vmno deployment.